### PR TITLE
Vue2: ready-replaced

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary
* ready-replaced

## Reviewer guidance

* Were all these `nextTick` to guarantee in-document, necessary?

## Issues addressed

* [ready-replaced](https://vuejs.org/v2/guide/migration.html#ready-replaced)

### CODE REVIEW

File: kolibri/core/assets/src/vue/core-modal/index.vue [hunk 1]
```diff
@@ -73,13 +73,15 @@
...
       },
     },
-    attached() {
+    mounted() {
       this.lastFocus = document.activeElement;
-      this.focusModal();
-      window.addEventListener('focus', this.focusElementTest, true);
+      this.$nextTick(() => {
+        this.focusModal();
+        window.addEventListener('focus', this.focusElementTest, true);
```
Comment (christianmemije) [inline kolibri/core/assets/src/vue/core-modal/index.vue]: can this be outside the nextTick?

Comment (indirectlylit) [inline kolibri/core/assets/src/vue/core-modal/index.vue]: probably not necessary to have the event listener inside `nextTick`

Comment (66eli77) [reply kolibri/core/assets/src/vue/core-modal/index.vue]: actually function  focusElementTest calls this.$refs.modal, so might be good to stay inside nextTick

Comment (66eli77) [reply kolibri/core/assets/src/vue/core-modal/index.vue]: ah. nvm focusElementTest actually checks if modal is in doc or not

**Pull request merged**